recipes/abacas/patches/path.patch

```diff
@@ -0,0 +1,8 @@
+--- abacas.1.3.1.pl	2010-11-03 09:30:45.000000000 -0500
++++ abacas.1.3.1.pl.new	2019-03-27 22:14:40.542475500 -0500
+@@ -1,4 +1,4 @@
+-#!/usr/local/bin/perl
++#!/opt/anaconda1anaconda2anaconda3/bin/perl
```

Comment:
You should use `#!/usr/bin/env perl`

Reply:
I think now it's correct, would you mind to check it please?